composer.json

```diff
@@ -12,8 +12,9 @@
   ],
   "require": {
     "php": ">=7.0",
+    "ext-json": "*",
     "composer/installers": "~1.0",
-    "ext-json": "*"
+    "woocommerce/action-scheduler": "^3.8"
```

Comment:
@delawski I think it'd be the same in the scheduled purges? It needs to run as a background process, I'm not sure we can account for anyone who's turned off cron or has such a small site that it doesn't run? I think maybe in that case there wouldn't be a lot of logs so it wouldn't run as a background process?

I'm not sure how to account for that case. We're using a recursive background process because it wouldn't necessarily work any other way so I don't think they'll be worse off. ?? That logic may be faulty.

Reply:
> That logic may be faulty.

No, I think your logic is correct. Only some very small (DIY?) hosting services would not have CRON running. I wouldn't assume such servers host sites that would require Stream at all.

Something else I could have considered before posting that question is that Woo itself relies on Action Scheduler and CRON. If they are okay with that choice, we should be too.